lib/addons/ack/index.ts

```diff
@@ -0,0 +1,108 @@
+// lib/certmanager_addon.ts
+import { ManagedPolicy } from 'aws-cdk-lib/aws-iam';
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace, setPath } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+/**
+ * User provided option for the Helm Chart
+ */
+export interface AckAddOnProps extends HelmAddOnUserProps {
+    
+    /**
+     * Name of the ack controller
+     * @default iam-chart
+     */
+    name?: string;

```

Comment:
Please remove all redundant attributes that are already declared in the HelmAddOnUserProps. Applies to name, chart, version, release, repository.

Reply:
Done.